modules/claims_api/config/initializers/okcomputer.rb

```diff
@@ -1,6 +1,6 @@
 # frozen_string_literal: true
 
-# require 'bgs/services'
+require 'bgs/services'
 require 'mpi/service'
 require 'evss/service'
 # require 'bgs_service/local_bgs'
```

Comment:
Looks like this was added in PR #12034. Left un-commented but not sure if we need this?